src/main/scala/org/broadinstitute/dsp/HelmInterpreter.scala

```diff
@@ -61,6 +61,23 @@ class HelmInterpreter[F[_]](concurrencyBound: Semaphore[F])(implicit logger: Log
       _ <- Kleisli.liftF(translateResult("helm list", "ok"))
     } yield ()
 
+  def updateAndPull(chartName: ChartName, chartVersion: ChartVersion, destDir: String): Kleisli[F, AuthContext, Unit] =
+    for {
+      ctx <- Kleisli.ask[F, AuthContext]
+      _ <- Kleisli.liftF(
+        boundF(
+          F.blocking(
+            helmClient.updateAndPull(
+              chartName,
+              chartVersion,
+              destDir
+            )
+          )
+        )
+      )
+      _ <- Kleisli.liftF(translateResult("helm updateAndPullChart", "ok"))
```

Comment:
nit: should this string be `updateAndPull`?  Or, alternately, should all these new methods you're adding be renamed to `updateAndPullChart`?

Reply:
Yeah, I did `updateAndPull` because it's shorter but I guess `updateAndPullChart` is a bit more descriptive.  I'll make it more consistent either way.

Reply:
+1 to `updateAndPullChart` everywhere :)